packages/cli/src/commands/deploy/index.ts

```diff
@@ -93,6 +93,7 @@ export default async (client: Client): Promise<number> => {
       '--prod': Boolean,
       '--archive': String,
       '--no-wait': Boolean,
+      '--no-alias': Boolean,
```

Comment:
Is "alias" the term we want to use here? I think it should match the name we call it in the dashboard and I dont think we have used "alias" in many years.

Reply:
I'm wondering the same. I am working on a discussion topic in Slack about this.